src/Domain/SelectedConfirmationPage.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\Domain;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class SelectedConfirmationPage {
```

Comment:
I think this makes most sense when regarded as part of the presentation layer, not the